<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brawl Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="gridContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6"/>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="bottomContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>280</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="p1Container" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="p1Box">
            <property name="maximumSize">
             <size>
              <width>155</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>P1</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="ClickableLabel" name="p1Portrait">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>98</height>
                </size>
               </property>
               <property name="statusTip">
                <string>Mario</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Resources.qrc">:/portraits/P_Mario.PNG</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="p1CharName">
               <property name="text">
                <string>Mario</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="p1SelectButton">
               <property name="statusTip">
                <string>Select which characters you want available.</string>
               </property>
               <property name="text">
                <string>Select Characters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="p1Check">
               <property name="statusTip">
                <string>Is this player in the match? Uncheck to take a break (or to freeze this slot while rerolling another).</string>
               </property>
               <property name="text">
                <string>Participating</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="p1StatsBox">
            <property name="statusTip">
             <string>The number of turns remaining until the player has used all characters once. See &quot;Character Reuse Rule&quot;.</string>
            </property>
            <property name="title">
             <string>Stats</string>
            </property>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Turns until Reset:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="p1ResetCountdown">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="p2Container" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="p2Box">
            <property name="maximumSize">
             <size>
              <width>155</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>P2</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="ClickableLabel" name="p2Portrait">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>98</height>
                </size>
               </property>
               <property name="statusTip">
                <string>Mario</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Resources.qrc">:/portraits/P_Mario.PNG</pixmap>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="p2CharName">
               <property name="text">
                <string>Mario</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="p2SelectButton">
               <property name="statusTip">
                <string>Select which characters you want available.</string>
               </property>
               <property name="text">
                <string>Select Characters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="p2Check">
               <property name="statusTip">
                <string>Is this player in the match? Uncheck to take a break.</string>
               </property>
               <property name="text">
                <string>Participating</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="p2StatsBox">
            <property name="statusTip">
             <string>The number of turns remaining until the player has used all characters once. See &quot;Character Reuse Rule&quot;.</string>
            </property>
            <property name="title">
             <string>Stats</string>
            </property>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Turns until Reset:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="p2ResetCountdown">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="p3Container" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="p3Box">
            <property name="maximumSize">
             <size>
              <width>155</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>P3</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="ClickableLabel" name="p3Portrait">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>98</height>
                </size>
               </property>
               <property name="statusTip">
                <string>Mario</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Resources.qrc">:/portraits/P_Mario.PNG</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="p3CharName">
               <property name="text">
                <string>Mario</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="p3SelectButton">
               <property name="statusTip">
                <string>Select which characters you want available.</string>
               </property>
               <property name="text">
                <string>Select Characters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="p3Check">
               <property name="statusTip">
                <string>Is this player in the match? Uncheck to take a break.</string>
               </property>
               <property name="text">
                <string>Participating</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="p3StatsBox">
            <property name="statusTip">
             <string>The number of turns remaining until the player has used all characters once. See &quot;Character Reuse Rule&quot;.</string>
            </property>
            <property name="title">
             <string>Stats</string>
            </property>
            <layout class="QFormLayout" name="formLayout_4">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Turns until Reset:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="p3ResetCountdown">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="p4Container" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="p4Box">
            <property name="maximumSize">
             <size>
              <width>155</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="title">
             <string>P4</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="ClickableLabel" name="p4Portrait">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>98</height>
                </size>
               </property>
               <property name="statusTip">
                <string>Mario</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="pixmap">
                <pixmap resource="Resources.qrc">:/portraits/P_Mario.PNG</pixmap>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="p4CharName">
               <property name="text">
                <string>Mario</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="Line" name="line_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="p4SelectButton">
               <property name="statusTip">
                <string>Select which characters you want available.</string>
               </property>
               <property name="text">
                <string>Select Characters</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="p4Check">
               <property name="statusTip">
                <string>Is this player in the match? Uncheck to take a break.</string>
               </property>
               <property name="text">
                <string>Participating</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="p4StatsBox">
            <property name="statusTip">
             <string>The number of turns remaining until the player has used all characters once. See &quot;Character Reuse Rule&quot;.</string>
            </property>
            <property name="title">
             <string>Stats</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Turns until Reset:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="p4ResetCountdown">
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="generalContainer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="generalBox">
            <property name="title">
             <string>General</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="rollButton">
               <property name="statusTip">
                <string>Pick characters for all participants!</string>
               </property>
               <property name="text">
                <string>Go!</string>
               </property>
               <property name="autoDefault">
                <bool>true</bool>
               </property>
               <property name="default">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearButton">
               <property name="statusTip">
                <string>Clear all selections</string>
               </property>
               <property name="text">
                <string>Clear</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Character Reuse Rule</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <widget class="QRadioButton" name="reuseRuleResetOne">
               <property name="statusTip">
                <string>When all of a player's characters have been used, make them all available again.</string>
               </property>
               <property name="text">
                <string>Reset One</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="reuseRuleResetAll">
               <property name="statusTip">
                <string>When any player runs out of characters, make all players' characters available again.</string>
               </property>
               <property name="text">
                <string>Reset All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="reuseRuleAllow">
               <property name="statusTip">
                <string>Don't disable characters at all. This means the same character can appear twice in a row.</string>
               </property>
               <property name="text">
                <string>Allow Reuse</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="resetNowButton">
               <property name="statusTip">
                <string>Reset all character queues</string>
               </property>
               <property name="text">
                <string>Reset Now</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header>ClickableLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
